src/main/kotlin/eu/europa/ec/eudi/openid4vci/DPoP.kt

```diff
@@ -143,14 +143,15 @@ fun HttpRequestBuilder.bearerOrDPoPAuth(
     htu: URL,
     htm: Htm,
     accessToken: AccessToken,
+    nonce: String?,
```

Comment:
I think that we need to keep to a single representation for (DPoP) Nonce
Options are:
1. Use a simple String
2. Typealias Nonce = String
3. Value class Nonce
4. Use of Nonce class from Nimbus

In the first PR, we used option 4, but probably it is not ideal to couple our public API to an external class.
I would go with the value class approach

Reply:
Introduce a new `Nonce` value class.